Lib/linecache.py

```diff
@@ -44,6 +45,22 @@ def getlines(filename, module_globals=None):
         return []
 
 
+def _getline_from_code(filename, lineno, module_globals=None):
```

Comment:
`module_globals` is never used and this is a new private entry, can we drop it off?